frontend-react/src/shared/Features/Feature.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
```

Comment:
I'm not sure I understand what a "feature" is here, nor what its usefulness is to the project as a whole. Also we're hardcoding a header size (h3) which makes this components' reusability lower.